src/figuresClasses.ts

```diff
@@ -1,11 +1,80 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number | string;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
+    public shape: string = 'triangle',
+  ) {
+    if (sideA <= 0 || sideB <= 0 || sideC <= 0) {
+      throw new Error('Arror');
```

Comment:
[CODE STYLE] - When throwing an error, provide a detailed description in the error message to help understand the reason for the error. For example, 'Side lengths must be greater than zero.'